ouroboros-consensus-diffusion/test/consensus-test/Test/Consensus/PeerSimulator/Run.hs

```diff
@@ -246,7 +246,9 @@ runPointSchedule schedulerConfig genesisTest tracer0 =
     resources <- makePeerSimulatorResources tracer gtBlockTree (getPeerIds gtSchedule)
     let getCandidates = traverse readTVar =<< readTVar (psrCandidates resources)
         getHandles = readTVar (psrHandles resources)
-        updateLoEFrag = updateLoEFragGenesis config (mkGDDTracerTestBlock tracer) getCandidates getHandles
+        getLatestSlots = traverse CSClient.cschLatestSlot =<< getHandles
```

Comment:
FTR: I think we additionally want to wake up the GDD when any of the CS clients (no longer) sends `MsgAwaitReply`.

Reply:
alright I'll look into it